ai-platform-unified/notebooks/custom_job_image_classification_model_for_online_prediction.ipynb

```diff
@@ -29,7 +29,7 @@
     "id": "eHLV0D7Y5jtU"
```

Comment:
<p>Python is misspelled</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>